contracts/services/template/src/msg.rs

```diff
@@ -53,4 +54,6 @@ impl OptionalServiceConfig {
 }
 
 #[cw_serde]
-pub struct Config {}
+pub struct Config {
+    pub admin: Addr,
+}
```

Comment:
why does Config need an admin now? I guess we can have an admin and an owner now and this admin is not updatable? I don't see this being used anywhere except the template now.